contracts/src/TeleporterTokenSource.sol

```diff
@@ -52,9 +51,9 @@ abstract contract TeleporterTokenSource is ITeleporterTokenBridge, TeleporterOwn
     constructor(
         address teleporterRegistryAddress,
         address teleporterManager,
-        address feeToken_
+        address feeTokenAddress_
     ) TeleporterOwnerUpgradeable(teleporterRegistryAddress, teleporterManager) {
-        feeToken = IERC20(feeToken_);
+        feeTokenAddress = feeTokenAddress_;
```

Comment:
Should we require that this is non-zero?

Reply:
yup nice catch, added some more base require checks and matching unit tests